apps/web/src/components/digest-demo-flow/DigestDemoFlow.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect } from 'react';
+import styled from '@emotion/styled';
+import { Skeleton, useMantineColorScheme } from '@mantine/core';
+import { useResizeObserver } from '@mantine/hooks';
+import ReactFlow, { Edge, Node, useReactFlow } from 'react-flow-renderer';
+
+import { colors } from '../../design-system';
+import { TriggerNode } from './TriggerNode';
+import { DigestNode } from './DigestNode';
+import { EmailNode } from './EmailNode';
+import { useTemplateFetcher } from '../../api/hooks';
+import { DigestDemoFlowProvider } from './DigestDemoFlowProvider';
+
+export function DigestDemoFlow({
+  isReadOnly = true,
+  templateId,
+  className,
+}: {
+  isReadOnly?: boolean;
+  templateId?: string;
+  className?: string;
+}) {
+  const [ref, rect] = useResizeObserver();
+  const { colorScheme } = useMantineColorScheme();
+  const reactFlowInstance = useReactFlow();
+  const { isInitialLoading: isLoadingTemplate } = useTemplateFetcher(
+    { templateId },
+    { enabled: !isReadOnly && !!templateId, refetchOnWindowFocus: false }
+  );
+
+  useEffect(() => {
+    reactFlowInstance.fitView({ minZoom: 1, maxZoom: 1 });
+  }, [rect.width, rect.height]);
+
+  return (
+    <DigestDemoFlowProvider isReadOnly={isReadOnly} templateId={templateId}>
```

Comment:
I've added the provider here that has the logic for trigger, interval and counts defined